builder/copy_unix_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) OpenFaaS Project 2017. All rights reserved.
```

Comment:
should just be `copy_test.go`

Reply:
I need to separate the declaration to aother file so that the declaration
is not redeclared in Windows.

Reply:
@itscaro thought about this at the weekend.. its just a test helper, why not just check for `runtime.GOOS` rather than split the helper into OS specific variants?

Reply:
@johnmccabe sounds good to me, thank you